test/evergreen/print_python_stack_trace.py

```diff
@@ -0,0 +1,129 @@
+    dash = "-"*count
+    return "+{dash}+\n| {msg} |\n+{dash}+".format(dash=dash, msg=msg)
+
+
+class LLDBDumper:
+    """LLDBDumper class."""
+    def __init__(self):
+        self.dbg = self.__find_debugger("lldb")
+
+    @staticmethod
+    def __find_debugger(debugger: str):
+        """Find the installed debugger."""
+        return which(debugger)
+
+    def dump(self, exe_path: str, core_path: str):
+        """Dump stack trace."""
+        if self.dbg is None:
+            sys.exit("Debugger gdb not found,"
+                     "skipping dumping of {}".format(core_path))
+
+        cmds = [
+            "thread backtrace all -c 30",
+            "quit"
+        ]
+
+        subprocess.run([self.dbg, "--batch"] + [exe_path, "-c", core_path ] +
+                        list(itertools.chain.from_iterable([['-o', b] for b in cmds])),
+                        check=True)
+
+
+class GDBDumper:
+    """GDBDumper class."""
+    def __init__(self):
+        self.dbg = self.__find_debugger("gdb")
+
+    @staticmethod
+    def __find_debugger(debugger: str):
+        """Find the installed debugger."""
+        return which(debugger)
+
+    def dump(self, exe_path: str, core_path: str, lib_path: str):
+        """Dump stack trace."""
+        if self.dbg is None:
+            sys.exit("Debugger gdb not found,"
+                     "skipping dumping of {}".format(core_path))
+
+        cmds = []
+        if lib_path:
+            cmds.append("set solib-search-path " + lib_path)
+        cmds.extend([
+            "thread apply all backtrace 30",
+            "quit"
+        ])
+
+        subprocess.run([self.dbg, "--batch", "--quiet"] +
+                        list(itertools.chain.from_iterable([['-ex', b] for b in cmds])) +
+                        [exe_path, core_path],
+                        check=True)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-e', '--executable_path',
+                        help='path to the executable',
+                        required=True)
+    parser.add_argument('-c', '--core_path',
+                        help='directory path to the core dumps',
+                        required=True)
+    parser.add_argument('-l', '--lib_path', help='library path')
+    args = parser.parse_args()
+
+    # Store the path of the core files as a list.
+    core_files = []
+    regex = re.compile(r'.*dump.*python.*', re.IGNORECASE)
+    for root, dirs, files in os.walk(args.core_path):
+        for file in files:
+            if regex.match(file):
+                core_files.extend([os.path.join(root, file)])
+
+    for core_file_path in core_files:
+        print(border_msg(core_file_path), flush=True)
+        if sys.platform.startswith('linux'):
+            dbg = GDBDumper()
+            dbg.dump(args.executable_path, core_file_path, args.lib_path)
+        elif sys.platform.startswith('darwin'):
+            dbg = LLDBDumper()
+            dbg.dump(args.executable_path, core_file_path)
+
```

Comment:
Let's add a comment here so people know Windows is in the works:
```
elif sys.platform.startswith('win32'):
    # FIXME - Windows to be supported in WT-XXXX
    pass
```

Reply:
Updated in [9f20287](https://github.com/wiredtiger/wiredtiger/pull/7645/commits/9f20287a1c6c6defe116476933be9e3c948bd697)

Reply:
Created [WT-8937](https://jira.mongodb.org/browse/WT-8937)